tools/dev/experiment/values-micro-services-experiment.yaml

```diff
@@ -0,0 +1,71 @@
+pyroscope:
+  extraEnvVars:
+    PYROSCOPE_V2_EXPERIMENT: 1
+  extraArgs:
+    query-backend.address: "dns:///_grpc._tcp.pyroscope-query-worker-headless.$(NAMESPACE_FQDN):9095"
+    metastore.address: "dns:///_grpc._tcp.pyroscope-metastore-headless.$(NAMESPACE_FQDN):9095"
+    metastore.raft.bind-address: ":9099"
+    metastore.raft.server-id: "$(POD_NAME).pyroscope-metastore-headless.$(NAMESPACE_FQDN)"
+    metastore.raft.advertise-address: "$(POD_NAME).pyroscope-metastore-headless.$(NAMESPACE_FQDN):9099"
+    metastore.raft.bootstrap-peers: "dnssrvnoa+_raft._tcp.pyroscope-metastore-headless.$(NAMESPACE_FQDN):9099"
+    metastore.raft.bootstrap-expect-peers: "3"
```

Comment:
This is why I added those env variables. Obviously, we can craft the addresses in the helm templates, but I decided not to modify them yet. I think that we may want to add component-specific templates instead of reusing the same one